components/ref/config.go

```diff
@@ -22,6 +22,12 @@ type Config struct {
 }
 
 type Item struct {
-	Name string `json:"name"`
-	Key  string `json:"key"`
+	//secret component name, such as : local.file
+	ComponentType string `json:"component_type"`
+	// key in the secret component
+	Key string `json:"key"`
+	//sub key in the secret component
+	SubKey string `json:"sub_key"`
+	// key need to inject into metadata
+	RefKey string `json:"ref_key"`
```

Comment:
How about call it `injected_as`, which is easy to understand

Reply:
you are right, bro